frontend/src/app/page.tsx

```diff
@@ -1,5 +1,36 @@
 'use client';
 
+import styled from 'styled-components';
+import Text from '../design-system/Text';
+
+const Wrapper = styled.main`
+  height: 100vh;
+  background-color: #1e293b;
+  color: #e2e8f0;
+  font-family: 'Geist Sans', sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
```

Comment:
why do we need color and font-family to styled wrapper?

Reply:
if you still need to use font family, use it from the styled component theme:
```suggestion
  height: 100vh;
  background-color: #1e293b;
  color: {({theme}) => theme.text.primary}
  font-family: {({theme}) => theme.font.primary}
  display: flex;
  justify-content: center;
  align-items: center;
`;
```